setup.py

```diff
@@ -1,4 +1,5 @@
 from setuptools import find_packages, setup
+from extras_require import extras_require
```

Comment:
I don't think I've seen a pattern where a standalone module is imported into `setup.py`, so not sure if this would work in general for building the library? Mostly I've seen all the boilerplate and utility functions defined directly in `setup.py`.